### DISCUSSION TITLE
server: support for non-gravatar avatars

### DISCUSSION THREAD

**Discussion by foospidy** [asciinema/discussions#203] [Ideas]
I'd like to be able to change my avatar, but I don't have/want a gravatar account.

**Comment by xgqfrms** [comment]
yeah, absolute should support user upload an image as the avatars.

**Comment by thesixthguest** [comment]
support uploading avatar

**Comment by onyxiarivera** [comment]
Can we please add this? It's literally been an open issue for 5 years.

**Comment by Cornelius-Figgle** [comment]
pls

**Comment by ghost** [comment]
Please, I don't think it's that difficult :)

**Comment by ku1ik** [comment]
No, it's not. But there are more valuable priorities than this.
